UnitTests/GitUITests/CommandsDialogs/FormBrowseTests.LeftPanel.Remotes.cs

```diff
@@ -183,12 +177,21 @@ private TreeNode GetRemoteNode(FormBrowse form)
             return remotesNode;
         }
 
-        private void RunFormTest(Action<FormBrowse> testDriver)
+        private void RunFormTest(Action<TreeNode> testDriver)
         {
             RunFormTest(
                 form =>
                 {
-                    testDriver(form);
+                    var remotesNode = GetRemoteNode(form);
+
+                    // wait up to 5 seconds until the remotes will have been loaded
+                    for (int i = 0; i < 50 && remotesNode.Nodes.Count != RemoteNames.Length; ++i)
+                    {
+                        Thread.Sleep(100);
+                        Application.DoEvents();
+                    }
```

Comment:
❕ Do not use this approach. Instead, directly wait for the specific background load operation to complete.